### DISCUSSION TITLE
希望 Windows 安装程序不需要管理员权限，或者是发布免安装版本

### DISCUSSION THREAD

**Discussion by DingJunyao** [he3-app/he3-feedback#158] [Q&A]
由于公司电脑限制，只能用非管理员权限的程序。He3 的安装程序需要管理员权限，因此无法安装使用。且没有免安装版本。

希望开发者能够解决上述问题。

- 对于安装，可以询问用户是为所有用户安装还是仅为自己安装。如仅为自己安装，可以安装到用户的 AppData 目录中（许多软件都有类似的例子）。
- 如果程序中有功能需要使用管理员权限，可以仅在使用对应功能时请求该权限。

顺便问一下，开发该软件时是否考虑了代理相关设置（或者是识别并使用系统代理）？公司网络需要设置代理才能访问外部网络，由于还未使用，我暂时不知道，敬请谅解。

**Comment by fcwys** [comment]
同希望推出绿色版

**Comment by lyzhang1999** [comment]
感谢您的建议，我们将尽快退出免安装版本解决需要管理员权限的问题，此外，关于代理问题，目前 He3 暂时没有单独设置代理的功能，默认使用系统网络，您可以为设备设置全局代理来实现。
